kernel/arch/dreamcast/hardware/pvr/pvr_dma.c

```diff
@@ -109,7 +112,7 @@ int pvr_dma_transfer(void *src, uintptr_t dest, size_t count, int type,
         return -1;
     }
 
-    pvr_dma[PVR_LMMODE0] = type == PVR_DMA_VRAM64 ? 0 : 1;
+    pvr_dma[PVR_LMMODE0] = type == PVR_DMA_VRAM32 ? 1 : 0;
```

Comment:
Are you 100% sure about this?  I think I played with changing it to but I didnt get better results.  I think PVR_DMA_VRAM64 is only specified when you trying to uploading texture data for the PVR.  Hard to believe its been wrong for so long but its definitely possible.

Reply:
`PVR_DMA_VRAM64`, `PVR_DMA_TA` and `PVR_DMA_YUV` - 64-bit
`PVR_DMA_VRAM32` - 32-bit

PVR_LMMODE0=0 is 64-bit (A + B 32-bit memory buses)
PVR_LMMODE0=1 use only one memory 32-bit bus

Reply:
For TA and YUV only one 32-bit bus may have been used by mistake.

Reply:
Oh! Okay. I thought wrong then. Any reason we are ever forced to use PVR_DMA_VRAM32?

Reply:
If someone want use TA Input together with texture memory transfer probably. Or YUV + texture.

Reply:
Sounds good.

Reply:
There must be another function here actually. Possibility of transfer via the Root bus instead of TA. Maybe need replace this one with it?